spinq10q/parameters.json

```diff
@@ -1,859 +1,402 @@
 {
-    "nqubits": 10,
-    "settings": {
-        "nshots": 1024,
-        "relaxation_time": 20000
-    },
-    "qubits": [
-        1,
-        2,
-        3,
-        4,
-        5,
-        6,
-        7,
-        8,
-        9,
-        10
+  "nqubits": 5,
+  "settings": {
+    "nshots": 1024,
+    "relaxation_time": 20000
+  },
+  "qubits": [
+    1,
+    2,
+    3,
+    4,
+    5
```

Comment:
Just to understand, if this configuration is for the first 5 qubits shouldn't it be from 0 to 4?
Or is there a qubit 0 which we are not considering

Reply:
Thanks for spotting this, solved in https://github.com/qiboteam/qibolab_platforms_qrc/pull/114/commits/e20735cbd0c2a7e74339664331a26d62b9ae5e39!

Reply:
si we enumerate qubits from 0 always?

Reply:
In programming, especially in Python, this convention is used.